x-pack/plugins/observability/public/components/threshold/i18n_strings.ts

```diff
@@ -20,7 +20,7 @@ export const LABEL_LABEL = i18n.translate(
 export const LABEL_HELP_MESSAGE = i18n.translate(
   'xpack.observability.threshold.rule.alertFlyout.customEquationEditor.labelHelpMessage',
```

Comment:
Make sure to rename the id or remove the related translations manually, as it is not clear when changing the `defaultMessage`, the related translations will be updated. ([link](https://elastic.slack.com/archives/C5TQ33ND8/p1688459235633549))